### ISSUE TITLE
Excess residual cytoplasm

### ISSUE THREAD

**Issue by MagoWyr** [obophenotype/human-phenotype-ontology#9833]
**Preferred term label:**
Excess residual cytoplasm

**Synonyms**

**Definition (free text, please give PubMed ID)**
"Excess residual cytoplasm is associated with abnormal spermatozoa produced from a defective spermiation process. This abnormal excess cytoplasm should not be confused with more physiological cytoplasmic residues that can be observed in direct microscopy of the ejaculate but not in dried, fixed and stained morphology smears." 
WHO laboratory manual for the examination and processing of human semen (6th Edition)

**Parent term (use hpo.jax.org/app)**
HP:0012864

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA: 399808

**Your nano-attribution (ORCID)**
0000-0002-7651-2403

**Comment by pnrobinson** [comment]
@MagoWyr 
can we label this
**Excess residual spermatozoal cytoplasm**
?

**Comment by MagoWyr** [comment]
Yes, that's fine

**Comment by pnrobinson** [comment]
New annotation
301106 
PMID:36481789   2/2

**Comment by pnrobinson** [comment]
added as HP:0034795
